### DISCUSSION TITLE
add column "Marking Code" in parts

### DISCUSSION THREAD

**Discussion by Muzeau** [Part-DB/Part-DB-server#278] [Ideas]
Hi,
It would be wise if we could search according to the "marking code" of many active SMD components. For this it would be convenient to have a column "marking code". Currently for this I use the MPN which is not obvious.

**Comment by jbtronics** [comment]
In principle it is possible to add another field like this. However I am not sure how helpful this is for filter purposes, as they are pretty short and often not really unique.

Besides that you can attach arbitary text data to parts, using the parameter system. Just add the info you want as a parameter to a part and put the info in the text field. In the table filters you can filter by content of this parameter
